### PR TITLE
Added new and renamed current Dynatrace plugin

### PR DESCRIPTION
**Pull request by johannes-b**
## Hey, I just made a Pull Request!

Dynatrace aims to provide two Dynatrace plugins, to provide the best experience for Backstage:Dynatrace users. The following mockup shows the update of the Backstage plugin directory once this PR is merged.

image

This PR includes: 
* Renaming for the current plugin from `Dynatrace` to `Dynatrace for Managed`
* Introduction of the new plugin, which is hosted here: https://github.com/Dynatrace/backstage-plugin
* Update of the icon.

#### :heavy_check_mark: Checklist

- [ ] ~~A changeset describing the change and affected packages. ([more info]~~(https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
  - No update of the documentation, but on the Backstage plugin directory. 
- [ ] ~~Tests for new functionality and regression tests for bug fixes~~
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/dynatrace-managed.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (vinzscam) [inline microsite/data/plugins/dynatrace-managed.yaml:1]: the content of this file looks like the same as `dynatrace.yaml`.
Is there any missing commit @johannes-b? 🤔

**Review by vinzscam — CHANGES_REQUESTED** [review]

File: microsite/data/plugins/dynatrace-managed.yaml (hunk 1 above)
Comment (johannes-b) [reply microsite/data/plugins/dynatrace-managed.yaml:1]: @vinzscam you are absolutely right. I forgot to bring the updated `dynatrace.yaml`. Please take a look now.

Comment (johannes-b) [reply microsite/data/plugins/dynatrace-managed.yaml:1]: @vinzscam can you please help me to figure out why the workflow is failing at `yarn install`? Unfortunately, I can`t figure out why my changes break the workflow.

Comment (vinzscam) [reply microsite/data/plugins/dynatrace-managed.yaml:1]: let's seeeee probably a bad merge. I've rebased and force pushed the branch

Comment (johannes-b) [reply microsite/data/plugins/dynatrace-managed.yaml:1]: Thanks for your help and force pushing the PR. After doing a `prettier:check` the PR passed the checks. I would appreciate a final review.

**Review by freben — APPROVED** [review]
Alright!

**Pull request merged**